README.md

```diff
@@ -8,7 +8,8 @@ Plugin for serving static files as fast as possible. Supports Fastify version `4
 
 | Fastify version | branch                                                               |
 | --------------- | -------------------------------------------------------------------- |
-| `^4.x`          | This branch                                                          |
+| `^5.x`          | This branch
```

Comment:
I see it did say `This branch` IMHO I think we should follow other repos like autoload [here](https://github.com/fastify/fastify-autoload?tab=readme-ov-file#compatibility)

```suggestion
| `^5.x`          | [`v8.x`](https://github.com/fastify/fastify-static/tree/master)
```

Reply:
Should I click on resolve or manually commit the change

Reply:
I think either or is fine